atlas/src/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { FallbackProps } from 'react-error-boundary'
+import { Button } from '@/shared/components'
+import { sizes, colors } from '@/shared/theme'
+
+const Container = styled.div`
+  padding: ${sizes.b4};
+  color: ${colors.gray[400]};
+  display: grid;
+  place-items: center;
+`
+
+const StyledButton = styled(Button)`
+  color: ${colors.white};
+`
+
+const ErrorFallback: React.FC<FallbackProps> = ({ error, resetErrorBoundary }) => {
+  return (
+    <Container>
+      <p>Something went wrong:</p>
+      <pre>{error?.message}</pre>
```

Comment:
I don't think we should present a JS error to the user

Reply:
You're right, I removed it and logged the error to the console in case we need it for debugging purposes.